samples/snippets/quickstart_analyzeiampolicylongrunning.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+
+from google.cloud import asset_v1
+
+
+def build_analysis_query(parent):
+    analysis_query = asset_v1.IamPolicyAnalysisQuery()
+    analysis_query.scope = parent
+    analysis_query.resource_selector.full_resource_name = "//cloudresourcemanager.googleapis.com/" + parent
+    analysis_query.options.expand_groups = True
+    analysis_query.options.output_group_edges = True
+
+    return analysis_query
+
+
+def analyze_iam_policy_longrunning_gcs(project_id, dump_file_path):
+    # [START asset_quickstart_analyze_iam_policy_longrunning_gcs]
+
+    # TODO project_id = 'Your Google Cloud Project ID'
+    # TODO dump_file_path = 'Your analysis dump file path'
+
+    client = asset_v1.AssetServiceClient()
+    parent = "projects/{}".format(project_id)
+    analysis_query = build_analysis_query(parent)
+    output_config = asset_v1.IamPolicyAnalysisOutputConfig()
+    output_config.gcs_destination.uri = dump_file_path
+    operation = client.analyze_iam_policy_longrunning(
+        request={"analysis_query": analysis_query, "output_config": output_config}
+    )
+
+    operation.result()
+    print(operation.done())
```

Comment:
Does it make sense to assert the success status?

Reply:
Getting result on line 48 is asserting success IMO. I may not get what you meant, could you provide a bit more details? Thank you.

Reply:
> Build failed due to the server didn't populate the response when done == true. I am fixing that and will pending this pull request on that fix.

Server side fixed. test can run successfully.